sphinx/source/text.rst

```diff
@@ -134,13 +134,13 @@ It should be noted that:
 - the order in which the flags are given does not change the result : ``!cl``
   will do just the same as ``!lc``.
 - Supplementarly exclamation marks will be ignored, and will not circumvent
-  the previuous rule : ``!l!c`` will do the same as ``!c!l`` or ``!cl``.
+  the previous rule : ``!l!c`` will do the same as ``!c!l`` or ``!cl``.
 
 The transformations are done in the following order:
 
 #. ``r``/``s`` (repr or str)
 #. ``t`` (translate)
-#. ``i`` (recursive interpolation)
+#. ``i`` (inner interpolation)
```

Comment:
This _is_ recursive. It doesn't stop at the first level.

Reply:
I had done that to have a word in "i", but "interpolation" is enough